package.json

```diff
@@ -119,17 +127,17 @@
   "husky": {
     "hooks": {
       "pre-commit": "pretty-quick --staged",
-      "pre-push": "make checkBindings",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
   },
   "prettier": "@stencila/dev-config/prettier-config.json",
   "release": {
-    "extends": "@stencila/semantic-release-config",
-    "message": "chore(release): ${nextRelease.version}\n\n${nextRelease.notes}"
+    "extends": "@stencila/semantic-release-config"
   },
   "dependencies": {
-    "astring": "^1.4.1",
-    "meriyah": "^1.6.10"
```

Comment:
Removing this dependency will break https://github.com/stencila/schema/blob/d9d275f334b61b3ee9051d210997e29081410b3c/ts/interpreter.ts#L3 this?